config/deploy.rb

```diff
@@ -0,0 +1,54 @@
+require 'dotenv'
+Dotenv.load
+
```

Comment:
I don't know if this is useful, but here's the deploy.rb file I use for another side project.  See anything we need from it?

```ruby
lock '3.4.0'

system "ssh-add ~/.ssh/id_rsa"

set :application, 'foobar'
set :deploy_to, '/home/foobar/app'
set :user, "datacenter"
set :use_sudo, false
set :scm, :git
set :repo_url, 'git@github.com:cantino/foobar.git'
set :pty, true
set :log_level, :debug

# Default branch is :master
ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }

set :bundle_without, [:development, :test]
set :bundle_binstubs, nil
set :bundle_flags, '--deployment --quiet --no-binstubs'

set :linked_dirs, fetch(:linked_dirs, []).push(*%w{log tmp/pids tmp/cache tmp/sockets public/system}) # vendor/bundle
set :linked_files, fetch(:linked_files, []).push(*%w{config/database.yml .env})

# Default value for default_env is {}
# set :default_env, { path: "/opt/ruby/bin:$PATH" }

after 'deploy:publishing', 'deploy:restart'

namespace :deploy do
  desc 'Restart the application services'
  task :restart do
    on roles(:app), in: :sequence, wait: 5 do
      sudo "stop #{fetch :application}" rescue nil
      sudo "start #{fetch :application}"
    end
  end

  desc 'Start the application services'
  task :start do
    on roles(:app), in: :sequence, wait: 5 do
      sudo "start #{fetch :application}"
    end
  end

  desc 'Stop the application services'
  task :stop do
    on roles(:app), in: :sequence, wait: 5 do
      sudo "stop #{fetch :application}"
    end
  end
end
```